### ISSUE TITLE
"The following tasks did not complete" using gulp 4 on Node 9

### ISSUE THREAD

**Issue by mgroenhoff** [gulpjs/gulp#2097]
This is caused by the fact that `gulp.dest(...)` never ends when writing a stream. See https://github.com/gulpjs/vinyl-fs/issues/295.

**Comment by MylesBorins** [comment]
@phated node team doesn't seem to care is pretty disingenuous and unfair imho. Single pings in a commit can be missed. Please open an issue next time. I'm digging in to see what's going on... seems like it might be an stream interface being used incorrectly in vinyl-fs

**Comment by phated** [comment]
@MylesBorins 
> pretty disingenuous and unfair imho

Yeah, maybe - but this is the umpteenth time we've been broken by something node core has done from upon high.  You're really the only one that has come in and helped explain where/why something was broken and submit the patches to fix it.  It just feels like there's 0 communication between core and userland (especially really big modules like us) and I already have too much on my plate to track commits that muck up FS WriteStream APIs that have been stable for 5+ years.

> Please open an issue next time.
Node's issue tracker is not an enjoyable place.

**Comment by phated** [comment]
For reference, this was fixed in vinyl-fs 3.0.2 - I'll get our CI matrices updated sometime in the future.
